XR_APILAYER_NOVENDOR_toolkit/layer.cpp

```diff
@@ -532,36 +543,27 @@ namespace {
 
         void updateStatisticsForFrame() {
             const auto now = std::chrono::steady_clock::now();
+            const auto numFrames = ++m_performanceCounters.numFrames;
 
-            m_frameTimestamps.push_back(now);
-
-            if (std::chrono::duration<double>(now - m_performanceCounters.lastWindowStart).count() > 1.0) {
-                // Update the FPS counter.
-                while (std::chrono::duration<double>(now - m_frameTimestamps.front()).count() > 1.0) {
```

Comment:
Great, this was remnant of when I updated the FPS "live" and not over 1s. This was not needed anymore, thanks for cleaning it up!